src/wtforms/validators.py

```diff
@@ -594,10 +594,20 @@ def __init__(self, values, message=None, values_formatter=None):
         self.values_formatter = values_formatter
 
     def __call__(self, form, field):
+        if field.type == "SelectMultipleField":
```

Comment:
This can't be the solution, as it excludes any other fields that have a list of data.

Could maybe do `if isinstance(field.data, list)`. That might have issues with `FieldList`, but that might not matter. Would also have issues if you want to check "none of these specific sets of choices", rather than "doesn't contain any of these choices".